<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BerechnungRC</class>
 <widget class="QWidget" name="BerechnungRC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RC_Berechnung</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureRC">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>35</y>
     <width>250</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Res.qrc">:/res/pictures/RLC.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="spannungTxt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Spannung U [V]</string>
   </property>
  </widget>
  <widget class="QLabel" name="frequenzTxt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequenz f [Hz]</string>
   </property>
  </widget>
  <widget class="QLabel" name="widerstandTxt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Widerstand R [Ohm]</string>
   </property>
  </widget>
  <widget class="QLabel" name="induktivitaetTxt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kapazit√§t C [F]</string>
   </property>
  </widget>
  <widget class="QLabel" name="eingabeTxt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eingabeparameter:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="spannungIn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="frequenzIn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="widerstandIn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="induktivitaet">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>pressed()</signal>
   <receiver>BerechnungRC</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
